src/gpodder/gtkui/shownotes.py

```diff
@@ -201,7 +201,14 @@ def update(self, heading, subheading, episode):
         else:
             self._base_uri = episode.channel.url
         self._loaded = False
-        self.html_view.load_html(episode.description_html, self._base_uri)
+
+        description = episode.description_html
+        if not description:
+            description = self.coerce_text_to_html(episode.description)
+        self.html_view.load_html(description, self._base_uri)
```

Comment:
Awesome, that sounds even better!

... I will let you try that though, because when I made the change, the description pane seemed to become stuck and not change to other descriptions :confused:

I have kept the logic you see above for the sake of being able to complete the merge (gpodder/gpodder#247), but welcome changes to it.